javascript/extractor/tests/flow/output/trap/anonFunctionWithoutParens.js.trap

```diff
@@ -72,20 +72,14 @@ locations_default(#20025,#10000,1,40,1,39)
 hasLocation(#20024,#20025)
 toplevels(#20001,0)
 hasLocation(#20001,#20003)
-#20026=@"module;{#10000},1,1"
-scopes(#20026,3)
-scopenodes(#20001,#20026)
-scopenesting(#20026,#20000)
-is_module(#20001)
-is_es2015_module(#20001)
-#20027=*
-entry_cfg_node(#20027,#20001)
-#20028=@"loc,{#10000},1,1,1,0"
-locations_default(#20028,#10000,1,1,1,0)
-hasLocation(#20027,#20028)
-#20029=*
-exit_cfg_node(#20029,#20001)
-hasLocation(#20029,#20025)
-successor(#20027,#20029)
+#20026=*
+entry_cfg_node(#20026,#20001)
+#20027=@"loc,{#10000},1,1,1,0"
+locations_default(#20027,#10000,1,1,1,0)
+hasLocation(#20026,#20027)
+#20028=*
+exit_cfg_node(#20028,#20001)
+hasLocation(#20028,#20025)
+successor(#20026,#20028)
```

Comment:
These files are written in a JS dialect called Flow (which is why they are in a folder called `flow`). We tolerate the presence of Flow type annotations and type alias declarations, but we don't extract them to the AST, which is why the `export type` declarations aren't seen by the new detector.

I'll think about the best course of action and follow up.

Reply:
Aha, that all makes more sense now :+1: 

> I'll think about the best course of action and follow up.

Sounds good :+1: